README.md

```diff
@@ -85,17 +85,18 @@ project.  Or, they write a dependency metadata file under `vendor/`
 which says how to get the dependencies.  But then anyone who wants to
 build the project needs to use a specific tool to retrieve the
 dependencies. (And there is no dominant standard for the dependency
-metadata files – there are even two different formats fore a file
+metadata files – there are even two different formats for a file
 called `vendor.json`.)
 
-Instead, vendetta relies on a feature of git:
-[submodules](https://git-scm.com/docs/git-submodule) provides a way
-for one git repository to point to another git repository (and a
-specific commit within it).  And submodules are a standard feature of
-git, so git will retrieve them for you.  You may already have
-experience with submodules.  And tools built on top of git understand
-submodules (e.g. github knows about submodules, and will display a
-submodule pointing to another project on github as a link).
+Instead, vendetta relies on the
+[submodule](https://git-scm.com/docs/git-submodule) feature of git,
+which provides a way for one git repository to point to another git
+repository (and a specific commit within it).  And submodules are a
+standard feature of git, so git will retrieve them for you.  You may
+already have experience with submodules.  And tools built on top of
+git understand submodules (e.g. github knows about submodules, and
+will display a submodule pointing to another project on github as a
+link).
```

Comment:
The issue here was the "submodules provides", which reads wrong - should either be "submodule provides" or "submodules provide", but both of these sound awkward in the context. Hence the more substantial rewording.